charts/jenkins/values.yaml

```diff
@@ -142,6 +142,10 @@ master:
   # When setting runAsUser to a different value than 0 also set fsGroup to the same value:
   runAsUser: 1000
   fsGroup: 1000
+  # If you have PodSecurityPolicies that require dropping of capabilities as suggested by CIS K8s benchmark, put them here
+  securityContextCapabilities: false
+  #  drop:
+  #    - NET_RAW
```

Comment:
which capabilities would you drop based on CIS benchmark? We could add all of them here in the comment.

Reply:
Actually, they suggest NET_RAW or ALL in chapter 5.2.7, and that's all they suggest. If we drop ALL, then we can't bind ports (need NET_BIND_SERVICE for that), so we're going with just NET_RAW in our setup. Alternative is to drop ALL and then add back the one's you need, but that requires extensive testing.